public/css/results_page.css

```diff
@@ -0,0 +1,23 @@
+#p1move{
+  animation:show 2s 2s;
+  animation-delay: 0ms;
+}
+
+#p2move{
+  color: transparent;
+  animation:show 2s 2s;
+  animation-fill-mode: forwards;
+  animation-delay: 2s;
+}
+
+#result{
+  color: transparent;
+  animation:show 2s 2s;
+  animation-fill-mode: forwards;
+  animation-delay: 4s;
+}
+
+@keyframes show{
```

Comment:
This feature is really cool and creative, it shows forward thinking and an awareness to the UI, making it more inviting to use. 
I like that you can approach this in different ways by using from/to or using percentages to control the action of the animation.